bbreadfront/src/SignUpForm.js

```diff
@@ -1,16 +1,19 @@
 import React, { useState } from "react";
 import { Redirect } from "react-router";
 function SignUpForm(props) {
+  
   const [person, setPerson] = useState({
     name: "",
     job: ""
   });
+  
   const [show, setShow] = React.useState(false);
   const [redirect, setRedirect] = React.useState(false);
   const [userData, setUserData] = useState({});
+  
   async function submitForm() {
     let signedIn = await props.handleSubmit(person);
-    console.log(signedIn)
```

Comment:
Thank for the refactoring of console logs and making sure the style is consistent. Looks good!

Reply:
Yes, of course. It just looks better now and that way we don't waste time looking for previous console.log that we forgot to delete.

Reply:
Looks good to me. And yeah the console logs look good